topics/templates/document-attributes-product.adoc

```diff
@@ -139,8 +139,9 @@
 :appserver_managementcli_link: https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/configuration_guide/#management_cli_overview
 :appserver_managementconsole_link: https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/configuration_guide/#management_console_overview
 
-:jdgserver_name: JDG
+:jdgserver_name: RHDG
 :jdgserver_version: 7.3.5
+:jdgserver_version_latest: 8.1
```

Comment:
```suggestion
:jdgserver_version_latest: 8.1.0
```

Reply:
this is about to change to 8.1.1. I think it should be 8.1